src/Swoole/SwooleExtension.php

```diff
@@ -15,7 +15,11 @@ class SwooleExtension
      */
     public function dispatchProcessSignal(int $processId, string $signal): bool
     {
-        return Process::kill($processId, $signal);
+        if (Process::kill($processId, 0)) {
```

Comment:
According to [the english docs](https://www.swoole.co.uk/docs/modules/swoole-process-kill), the second parameter expects a string and is optional. Why pass `0` here?

Reply:
There are some problems with the English document. I've contacted to modify it.

Kill 0 is used to detect whether the process exists.

Reply:
The [PHP extension docs](https://www.php.net/manual/en/swoole-process.kill.php) show the same, although I've no idea how and from what these are generated. Just pointing it out though, not saying yours is wrong.